### ISSUE TITLE
Missing artifact comb.github.ccob:bittrex4j:jar:0.0.1-SNAPSHOT

### ISSUE THREAD

**Issue by CryptoTropico** [CCob/bittrex4j#1]
Hey man, first of all thank you so much for this!

I've tried using the Sonatype Nexus repository with Maven (in Eclipse IDE) and it gives me this error:

`Missing artifact comb.github.ccob:bittrex4j:0.0.1-SNAPSHOT`
`Failure to find comb.github.ccob:bittrex4j:0.0.1-SNAPSHOT in https://oss.sonatype.org/content/repositories/snapshots/`

Is the repo up?

As an alternative, it would be really helpful if you could provide a list of dependencies...

I've been nailing them one by one, and so far, I've worked out these:

> jackson
> gson
> slf4j
> microsoft.aspnet.signalr
> jaxb-api
> http.client
> http.core
> http.commons.codec

But I seem to be having an issue with http.core and http.client ... seems like theire not the right ones.

So, I'm stuck. Haven't been able to run any of the tests!

Thanks mate

**Comment by CCob** [comment]
The repo is moving quite fast right now.  I'm trying to get a maven release out via Sonatype which will be 1.0.0 but their end is holding this up.  There is a 1.0.1-SNAPSHOT available right now.  One thing I've noticed that's not on the README is the location where some of the dependencies are.  I'll update that when I can but in the mean time if you use these repos all should be fine.

```
 mavenCentral()
 maven { url "https://clojars.org/repo" }
 jcenter()
```

These are through gradle.  I'm afraid I don't know the actual URL for jcenter if you are using maven and pom.xml

**Comment by CCob** [comment]
I've also just noticed a typo in the README.md for the groupId, it should be `com.github.ccob`, doh.  I have updated the README to reflect this.  Let me know if this works and I can close the issue

**Comment by CryptoTropico** [comment]
Thank you!

I updated the repository and dependencies in Maven as you indicate in the updated README.md and it downloaded all my dependencies without any issues.

Now, there's an error when I try to run ShowRealTimeFills.java with this set of dependencies:

> Exception in thread "main" java.lang.NoClassDefFoundError: org/java_websocket/drafts/Draft
> 	at donky.microsoft.aspnet.signalr.client.transport.AutomaticTransport.initialize(AutomaticTransport.java:62)
> 	at donky.microsoft.aspnet.signalr.client.transport.AutomaticTransport.<init>(AutomaticTransport.java:44)
> 	at donky.microsoft.aspnet.signalr.client.Connection.start(Connection.java:261)
> 	at com.github.ccob.bittrex4j.BittrexExchange.connectToWebSocket(BittrexExchange.java:149)
> 	at com.github.ccob.bittrex4j.samples.ShowRealTimeFills.main(ShowRealTimeFills.java:28)
> Caused by: java.lang.ClassNotFoundException: org.java_websocket.drafts.Draft
> 	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
> 	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
> 	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)

Not sure what's going on...!

**Comment by CCob** [comment]
That's a transitive dependency of the signalr client library.  https://mvnrepository.com/artifact/org.java-websocket/java-websocket/1.3.1

This is the one from the clojars repo, not sure why it's not on your class path.  Are you using maven or gradle.  I'm using gradle and it imports it and applies it to the class path without issue

**Comment by CCob** [comment]
I'll close this now since I have released 1.0.3 which now depends on a forked version of signalr which it's dependencies have been upgraded and are all on Maven Central.  So no need for clojars or jcenter
